### DISCUSSION TITLE
📃 Triage Support ~ Top Known Issues

### DISCUSSION THREAD

**Discussion by MariMakes** [Ultimaker/Cura#16440] [Collaboration]
Hey All 👋 ,

Since searching for duplicates on GitHub is quite tricky, I've compiled a list of the issues which I believe are reported often. This is not a complete list, but one that I believe to be most helpful. If you feel that there are things clearly missing here,  you can leave a comment and I can update this overview to reflect it. 

_Last Updated 09-08-2023_

Crashes need to be evaluated on a case-by-case basis. We don’t know if it’s a duplicate until we’ve had a developer look at it with the engine in debug mode, so I left them out of the overview. 

**Trending Issues**
- Missing printer-related definitions, specifically Elegoo? 
https://github.com/Ultimaker/Cura/issues/16007

- Support Brim is missing if buildplate adhesion is set to none 
https://github.com/Ultimaker/Cura/issues/14983

- Supportbrim is printed before Skirt 
https://github.com/Ultimaker/Cura/issues/15088

- Mac Installer has a damaged "bundle-identifier" (CFBundleIdentifier)  
https://github.com/Ultimaker/Cura/issues/14375

- Align to Buildplate doesn’t work for grouped meshes
https://github.com/Ultimaker/Cura/issues/14237

- Weird small brims in dual extrusion
https://github.com/Ultimaker/Cura/issues/16055

- Unable to click multiple files on Windows
https://github.com/Ultimaker/Cura/issues/15879

- Brim Distance not working anymore for Per Object Settings
https://github.com/Ultimaker/Cura/issues/14862

- Enable Brim Only on Inside as an option 
https://github.com/Ultimaker/Cura/issues/2853

- Trouble with opening STLs
https://github.com/Ultimaker/Cura/issues/16249

**Commonly Reported issues**
1. Multiple Monitor Issue
https://github.com/Ultimaker/Cura/issues/11907

2. Profiles missing when creating a new material, not based on a generic material
https://github.com/Ultimaker/Cura/issues/8229

3. Improve Klipper support
https://github.com/Ultimaker/Cura/discussions/16012

4. Auto Arrange focuses more on placing big objects in the center rather than making sure everything fits
https://github.com/Ultimaker/Cura/issues/14112

5. Issues with scrolling behavior
https://github.com/Ultimaker/Cura/issues/12110

6. Cura Doesn't remember last-opened file directory.
https://github.com/Ultimaker/Cura/issues/12295

7. Combing Not In Skin is broken
https://github.com/Ultimaker/Cura/issues/14511

8. Hide seam is actually not hidden and gives random results
Could you check if you get the same result if you change the Seam Corner Preference to None?
https://github.com/Ultimaker/Cura/issues/15290

9. Field Loses focus while reducing a value to 0 to type
https://github.com/Ultimaker/Cura/issues/12949

10. Raft top layer incomplete on certain positions on the buildplate
 https://github.com/Ultimaker/Cura/issues/12276

When I'm not sure if someone's Operating System is still supported, I often forward them to this page:
https://support.makerbot.com/s/article/1667337917781

@fieldOfView , @GregValiant , @Asterchades, I believe this list could help you out the most. 

Again, feel free to use this comment section so we can start adding and/or removing things from this list.

**Comment by Ergonomicmike** [comment]
According to @GregValiant , there is a problem with slicing that started in Cura 5.
[Strange extrusions in 'Taco Bell Bag' ("Vase" Mode)](https://github.com/Ultimaker/Cura/issues/16412) #16412

**Comment by 5axes** [comment]
It would also be good to do the same with the most frequently requested **Enhancements** / **New Feature** . ;)

**Comment by GregValiant** [comment]
There may be something going on between Cura and Windows 11.  There are a lot of problems being reported in regards to "Won't Slice" with Win11 but I'm able to slice the models in Win10.

**Comment by 5axes** [comment]
**CURA 5.6 Trending Issues**

1 - Slice Failed due to **Print Thin Walls**

https://github.com/Ultimaker/Cura/issues/17897
https://github.com/Ultimaker/Cura/issues/17863
https://github.com/Ultimaker/Cura/issues/17853
https://github.com/Ultimaker/Cura/issues/17828
https://github.com/Ultimaker/Cura/issues/17750
https://github.com/Ultimaker/Cura/issues/17725
https://github.com/Ultimaker/Cura/issues/17679
https://github.com/Ultimaker/Cura/issues/17676
https://github.com/Ultimaker/Cura/issues/17671
https://github.com/Ultimaker/Cura/issues/17606
https://github.com/Ultimaker/Cura/issues/17560
https://github.com/Ultimaker/Cura/issues/17528
https://github.com/Ultimaker/Cura/issues/17503
https://github.com/Ultimaker/Cura/issues/17499
https://github.com/Ultimaker/Cura/issues/17470
https://github.com/Ultimaker/Cura/issues/17467
https://github.com/Ultimaker/Cura/issues/17457
https://github.com/Ultimaker/Cura/issues/17456
https://github.com/Ultimaker/Cura/issues/17496
https://github.com/Ultimaker/Cura/issues/17478
https://github.com/Ultimaker/Cura/issues/17439
https://github.com/Ultimaker/Cura/issues/17438
https://github.com/Ultimaker/Cura/issues/17410
https://github.com/Ultimaker/Cura/issues/17367
https://github.com/Ultimaker/Cura/issues/17344
https://github.com/Ultimaker/Cura/issues/17177
https://github.com/Ultimaker/Cura/issues/16688
https://github.com/Ultimaker/Cura/issues/16515
https://github.com/Ultimaker/Cura/issues/16417

For some geometries, activation of the Print Thin Walls option generates an error that stops the G-Code generation process..

2 - Slice Failed on **Windows 11**

https://github.com/Ultimaker/Cura/issues/17906
https://github.com/Ultimaker/Cura/issues/17895
https://github.com/Ultimaker/Cura/issues/17818
https://github.com/Ultimaker/Cura/issues/17801
https://github.com/Ultimaker/Cura/issues/17798
https://github.com/Ultimaker/Cura/issues/17790
https://github.com/Ultimaker/Cura/issues/17787
https://github.com/Ultimaker/Cura/issues/17775
https://github.com/Ultimaker/Cura/issues/17675
https://github.com/Ultimaker/Cura/issues/17659
https://github.com/Ultimaker/Cura/issues/17645
https://github.com/Ultimaker/Cura/issues/17636
https://github.com/Ultimaker/Cura/issues/17635
https://github.com/Ultimaker/Cura/issues/17617
https://github.com/Ultimaker/Cura/issues/17613
https://github.com/Ultimaker/Cura/issues/17612
https://github.com/Ultimaker/Cura/issues/17595
https://github.com/Ultimaker/Cura/issues/17571
https://github.com/Ultimaker/Cura/issues/17567
https://github.com/Ultimaker/Cura/issues/17566
https://github.com/Ultimaker/Cura/issues/17552
https://github.com/Ultimaker/Cura/issues/17546
https://github.com/Ultimaker/Cura/issues/17538
https://github.com/Ultimaker/Cura/issues/17530
https://github.com/Ultimaker/Cura/issues/17526
https://github.com/Ultimaker/Cura/issues/17523
https://github.com/Ultimaker/Cura/issues/17509
https://github.com/Ultimaker/Cura/issues/17491
https://github.com/Ultimaker/Cura/issues/17490
https://github.com/Ultimaker/Cura/issues/17461  ??
https://github.com/Ultimaker/Cura/issues/17380
https://github.com/Ultimaker/Cura/issues/17343
https://github.com/Ultimaker/Cura/issues/17335
https://github.com/Ultimaker/Cura/issues/17253

Recurring cases of slicing issues on Windows 11 which, when tested on Windows 10 workstations, generate no error.

3 - Slice Failed due to **Tree Support** 

https://github.com/Ultimaker/Cura/issues/17905
https://github.com/Ultimaker/Cura/issues/17898
https://github.com/Ultimaker/Cura/issues/17875
https://github.com/Ultimaker/Cura/issues/17829
https://github.com/Ultimaker/Cura/issues/17816
https://github.com/Ultimaker/Cura/issues/17811
https://github.com/Ultimaker/Cura/issues/17807
https://github.com/Ultimaker/Cura/issues/17688
https://github.com/Ultimaker/Cura/issues/17670
https://github.com/Ultimaker/Cura/issues/17655
https://github.com/Ultimaker/Cura/issues/17591
https://github.com/Ultimaker/Cura/issues/17597
https://github.com/Ultimaker/Cura/issues/17582
https://github.com/Ultimaker/Cura/issues/17578
https://github.com/Ultimaker/Cura/issues/17575
https://github.com/Ultimaker/Cura/issues/17549
https://github.com/Ultimaker/Cura/issues/17529
https://github.com/Ultimaker/Cura/issues/17521
https://github.com/Ultimaker/Cura/issues/17520
https://github.com/Ultimaker/Cura/issues/17479
https://github.com/Ultimaker/Cura/issues/17412
https://github.com/Ultimaker/Cura/issues/17351
https://github.com/Ultimaker/Cura/issues/17331

With Cura versions 5.5 and 5.6, if the settings do not allow tree-shaped supports to reach the supported area (branch angle, trunk diameter, space to create the support), the software generates an error and does not continue the slicing operation. Expecting result : No support if they cannot be generate but a model sliced.


4 - Slicing success can be Part position dependant 

https://github.com/Ultimaker/Cura/issues/17904
https://github.com/Ultimaker/Cura/issues/17853
https://github.com/Ultimaker/Cura/issues/17697
https://github.com/Ultimaker/Cura/issues/17682
https://github.com/Ultimaker/Cura/issues/17670
https://github.com/Ultimaker/Cura/issues/17610
https://github.com/Ultimaker/Cura/issues/17598
https://github.com/Ultimaker/Cura/issues/17494
https://github.com/Ultimaker/Cura/issues/17473
https://github.com/Ultimaker/Cura/issues/17466

Sometimes changing the position or rotation of a part can make or break the cut.

5- First Layer Wallspeed too fast

https://github.com/Ultimaker/Cura/issues/17594
https://github.com/Ultimaker/Cura/issues/17389
https://github.com/Ultimaker/Cura/issues/17010

If we have just some Wall on sliced geometry initial speed are not printed at initial layer speed

6- FLSUN 400 + Windows

https://github.com/Ultimaker/Cura/issues/17683
https://github.com/Ultimaker/Cura/issues/17645
https://github.com/Ultimaker/Cura/issues/17640

Several issues and when we try to load the sample project Cura Crash on Windows 10

7 - Slicing fails with CuraEngine Tiled Infill Generation

https://github.com/Ultimaker/Cura/issues/17739
https://github.com/Ultimaker/Cura/issues/17551

Slicing fails with CuraEngine Tiled Infill Generation 0.2.0 Plugin Enabled on Linux OS


Last problem for versions 5.5 & 5.6 the slicing time in some cases exploded. I didn't see many report on the github but some exchanges on the forum : https://community.ultimaker.com/topic/44443-slicing-stalls-in-cura-55/#comment-327364

And

https://github.com/Ultimaker/Cura/issues/17555
https://github.com/Ultimaker/Cura/issues/17726

**Comment by GregValiant** [comment]
@5axes - nice piece of work there.

**Comment by 5axes** [comment]
Some complementary tests and data concerning slicing times between different slicer versions for parameters as similar as possible. 
Tests made on Windows 10

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/lalliard/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/lalliard/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">


  | nasa-fabric-12x16 | nasa-fabric-12x16 Splited | Cave Entrance | Iron_Mask_OnePiece | Dragon Pencil Cup
-- | -- | -- | -- | -- | --
Cura 4.13.1 (Not Arachne   engin) | 1 mn 53s |   | 2 mn 18 s |   | 1 mn 51 s
Cura 4.20.14 (Not Arachne   engin) | 2 mn 41 s |   | 1 mn 43 s |   | 1 mn 32 s
Cura 5.4 | 2 mn 16 s | 1 mn 12 s | 47 s |   | 1 mn 3 s
Cura 5.5 | 14 mn 40 s | 1 mn 21 s | 1 mn 37 s |   | 2 mn 17 s
Cura 5.6 | 15 mn 56 s | 1 mn 23 s | 1 mn 37 s | 6 mn 59 s | 2 mn 05 s
PrusaSlicer  2.7.0 | 49 s | 57 s | 1 mn 03 s | 1 mn 38 s | 22 s
SuperSlicer    2.5.59.2 | 56 mn 02 s | 53 s | 52 s (Ajusted   Support) | 4 mn 13 s | 54 s    (Ajusted Support)
Orca Slicer  1.8.0 | 4 mn 45 s | 1 mn 03 s | 52 s (Organic   Support) | 3 mn 58 s | 49 s (Organic Support)



</body>

</html>

**Comment by 5axes** [comment]
Special mention for the [Great_Lion_Wall_Light](https://github.com/Ultimaker/Cura/files/12363064/Great_Lion_Wall_Light_0d33f4a0ca.zip)

![image](https://github.com/Ultimaker/Cura/assets/11015345/706b7f41-2f58-4a71-a3be-7de1fb9034b0)

At least **6 reports** for this part a record in the category.

https://github.com/Ultimaker/Cura/issues/17956
https://github.com/Ultimaker/Cura/issues/17530
https://github.com/Ultimaker/Cura/issues/17310
https://github.com/Ultimaker/Cura/issues/16505
https://github.com/Ultimaker/Cura/issues/16497
https://github.com/Ultimaker/Cura/issues/16366
